### PR TITLE
[release] convert tune release tests

### PR DESCRIPTION
**Pull request by krfricke**
## Why are these changes needed?

cc @richardliaw @amogkam 

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by krfricke** [comment]
This should be ready for review cc @richardliaw @amogkam

**Comment by krfricke** [comment]
Note: I ran the release tests for 1.4.0 on this branch (and they passed)

File: release/long_running_tests/workloads/impala.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import ray
 from ray.tune import run_experiments
-from ray.cluster_utils import Cluster
+import os
```
Comment (richardliaw) [inline release/long_running_tests/workloads/impala.py:5]: cc @sven1977 can you sign off on this?

Comment (sven1977) [reply release/long_running_tests/workloads/impala.py:5]: Looks good to me.

Comment (sven1977) [reply release/long_running_tests/workloads/impala.py:5]: @richardliaw ^

File: release/long_running_tests/workloads/impala.py [hunk 2]
```diff
@@ -16,19 +16,23 @@
...
+#     cluster.add_node(
+#         redis_port=6379 if i == 0 else None,
+#         num_redis_shards=num_redis_shards if i == 0 else None,
+#         num_cpus=10,
+#         num_gpus=0,
+#         resources={str(i): 2},
+#         object_store_memory=object_store_memory,
+#         redis_max_memory=redis_max_memory,
+#         dashboard_host="0.0.0.0")
+# ray.init(address=cluster.address)
```
Comment (richardliaw) [inline release/long_running_tests/workloads/impala.py:31]: I'm not quite sure about this change, but I'll open an issue to track this.

**Review by richardliaw — APPROVED** [review]

**Pull request merged**